src/main/java/lukemccon/airdrop/controllers/PackageTabCompletion.java

```diff
@@ -5,22 +5,35 @@
 import org.bukkit.command.CommandSender;
 import org.bukkit.command.TabCompleter;
 
+import java.sql.Array;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
 
 public class PackageTabCompletion implements TabCompleter {
     @Override
     public List<String> onTabComplete(CommandSender commandSender, Command command, String alias, String[] args) {
 
         // args[0] verified to be "package"
 
+        switch (args.length) {
 
-        // Return a list of packages
-        if (args.length == 2) {
-            return PackageManager.getPackages().stream().toList();
-        } else {
-            return new ArrayList<String>();
+            case 2:
+                List<String> commands = new ArrayList<>(PackageManager.getPackages().stream().toList());
+                commands.add("create");
+                commands.add("delete");
+                return commands;
+            case 3:
+                if (args[1].equals("create") || args[1].equals("delete")) {
+                    return new ArrayList<>(List.of("[packageName]"));
+                }
+            case 4:
+                if (args[1].equals("create")) {
+                    return new ArrayList<>(List.of("[price]"));
+                }
+            default:
+                return new ArrayList<String>();
```

Comment:
Assuming the returned list is never edited, we should use ```Collections.emptyList()``` here. Its faster and very clear that its just an empty list.